src/transforms.jl

```diff
@@ -273,10 +283,10 @@ function applyfeat(transform::Colwise, feat, prep)
   newfeat = 𝒯 |> Tables.materializer(feat)
 
   # cache values for each column
-  fcache = last.(vals)
```

Comment:
Isn't fcache the correct term in this context? We are inside the applyfeat, right?

Reply:
In this case, fcache is the tuple of colcaches and snames:
```julia
fcache = (caches, snames)
```